scripts/create_uber_extension_project.ts

```diff
@@ -250,7 +250,7 @@ function parseAndValidateArgs() {
   return { source, target, project, connection }
 }
 
-function isValidDirectory(name) {
+function isValidDirectory(name: string) {
```

Comment:
Did you fix, or did the linter add `: string`?

Reply:
I fixed it. These need to be linted as well. Another ticket